### PR TITLE
fix bug in HMM_align & add HMM-VAE example

### PR DESCRIPTION
**Pull request by Baileyswu**
fix bug:
 module 'beer' has no attribute 'VariationalBayesOptimizer'

### CODE REVIEW

**Comment by iondel** [comment]
First of all, thanks a lot for making this PR, that's very useful. 

A few things needs to be changed before I can merge it to the master branch:
  1. to make the training working, you should not backpropagate the forward-backward algorithm of the HMM. Practically, at https://github.com/Baileyswu/beer/blob/master/beer/models/hmm.py#L80 replace `pc_llhs` to `pc_llhs.detach()` to cut the gradient computation at this point
  2. looking at https://github.com/Baileyswu/beer/blob/master/beer/models/vae.py#L80 it seems you ignore the decoder completely...  
  3. Could you change you `HMM-VAE``script to a jupyter notebook ? It's easier for people to run it step by step.

**Comment by iondel** [comment]
I tried to run the code after your last update and the ELBO is increasing as it should be ! I'd be more than happy to make it into the master. BTW, there is still a hard-coded path into your VAE-HMM script. 
Otherwise, would you be able to make it in a jupyter notebook ?

**Comment by Baileyswu** [comment]
Thanks for your tutorial and attention, I have written a jupyter notebook at the same time. As the *.py is easy to debug step by step, I modify it first, and then I will upload .ipynb as you expect. Sorry for keeping you waiting for a long time because I have  been learning pytorch in [this repo](https://github.com/Baileyswu/pytorch-exercise), by which I can understand your bug reports. Thank you anyway!

**Comment by Baileyswu** [comment]
too many personal settings in this branch. delete redundant files in #85
